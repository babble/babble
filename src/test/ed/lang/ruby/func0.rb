
def simple()
  print( "yo" );
end

simple();

def welcome(name)
  print( "howdy" + name );
  puts "howdy #{name}"
end
welcome("eliot");
