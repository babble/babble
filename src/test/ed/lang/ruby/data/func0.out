yo
yo
howdyeliot
howdy eliot

hhh
